### ISSUE TITLE
Custom CSS support

### ISSUE THREAD

**Issue by raveenak28** [trgkanki/remaining_time#15]
Updated the addon while changing to 2.1.29 from 2.1.6 and the text size of the addon is significantly larger than it was before.
Any way to decrease the size?

**Comment by phu54321** [comment]
This should rather be done with custom CSS. Could you post your screenshot?

**Comment by ayynonwhymouse** [comment]
Came here to ask the same question. I just found this add-on, it is AMAZING. But the font does look too large & out of place for me as well. 

![anki_5ZwAbLWGpc](https://user-images.githubusercontent.com/67765070/96840893-af6b2900-1410-11eb-9f38-f1faba3f580a.png)

Can you point me in the right direction of how to add custom CSS? I have only the slightest knowledge of that sort of thing from editing a few other addons. But I glanced at the folders for this one and only saw python and json files.

**Comment by phu54321** [comment]
ETA today. With the next update, try this addon setting like:

```
{
    // ... maybe other things here...
    "barCSS": ".rt-message { text-size: .8em; }",
    // ... maybe other things here...
}
```

**Comment by phu54321** [comment]
Committed & released with v20.10.21i160
